tests/service/mod.rs

```diff
@@ -239,8 +239,23 @@ impl Service {
 
         for ip in listen_ips.clone() {
             let server = Server::new(ztauthority.to_owned());
+
+            let (tcp_socket, udp_socket) = match Server::bind(ip.ip()).await {
+                Ok(x) => x,
+                Err(_) => {
+                    panic!("Could not bind port");
```

Comment:
might be better to change the sig of the return value to return `Result<Vec<_>, anyhow::Error>` so the original error doesn't get swallowed.

Reply:
Took a stab at that but then I'd have to change the signature `Service::new` and then of every test, so...
People running zeronsd tests can figure it out :)

Reply:
I'm not sure that's the best way, but I won't argue with you. :P